content/blog/2019-03-07-commcall-mar2019.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: commcall-mar2019
+title: Community Call - Research Applications of rOpenSci Taxonomy and Biodiversity Tools
+date: '2019-03-07'
+authors:
+  - Stefanie Butland
+categories: blog
+topicid:
+tags:
+  - community
+  - events
+  - community call
+  - taxonomy
+  - biodiversity
+  - antarctic
+  - taxize
+  - taxizedb
+  - rgbif
+  - spocc
+  - taxa
+  - metacoder
+  - worrms
+  - wikitaxa
+  - taxview
+  - taxizesoap
```

Comment:
can remove `taxizesoap`. it's more or less abandoned

Reply:
done